public/ballers_new.sql

```diff
@@ -42,34 +42,450 @@ CREATE TABLE `area` (
+(154, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 4500000, 'Nigerian Property Centre.com', 'https://nigeriapropertycentre.com/for-rent/houses/terraced-duplexes/lagos/lekki/ikate-elegushi/602154-fully-serviced-4-bedroom-terrace-duplex'),
+(155, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 5000000, 'Nigerian Property Centre.com', 'https://nigeriapropertycentre.com/for-rent/houses/terraced-duplexes/lagos/lekki/ikate-elegushi/623452-fully-serviced-4-bed-terrace-duplex-with-bq'),
+(156, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 6500000, 'Nigerian Property Centre.com', 'https://nigeriapropertycentre.com/for-rent/houses/terraced-duplexes/lagos/lekki/ikate-elegushi/613386-newly-built-service-4-bedroom-terrace-duplex-in-a-serene-and-secured-n'),
+(157, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 3000000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-serviced-terrace-for-rent-kunsela-road-ikate-elegushi-sdly1338'),
+(158, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 3500000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-serviced-duplex-for-rent-ikate-ikate-elegushi-jast89740'),
+(159, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 4000000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-semi-detached-serviced-duplex-for-rent-ikate-ikate-elegushi-htpl0154'),
+(160, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 4500000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-serviced-terrace-for-rent-innocent-idibia-ikate-elegushi-sdly1197'),
+(161, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 5000000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-serviced-terrace-for-rent-off-elegushi-road-ikate-elegushi-sdly1156'),
+(162, 'For Rent', 'Ikate', 3, 1, '4 terrace duplex', 6000000, 'Private Property Nigeria.com', 'https://www.privateproperty.com.ng/listings/4-bed-serviced-terrace-for-rent-jakande-ikate-elegushi-sdly1234');
 
-CREATE TABLE `houses` (
-  `house_id` int(255) NOT NULL,
-  `type` varchar(255) NOT NULL,
-  `price` varchar(255) NOT NULL,
-  `area_id` int(255) NOT NULL,
-  `state_id` int(255) NOT NULL,
-  `longitude` varchar(255) DEFAULT NULL,
-  `latitude` varchar(255) DEFAULT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+-- --------------------------------------------------------
 
 --
--- Dumping data for table `houses`
---
-
-INSERT INTO `houses` (`house_id`, `type`, `price`, `area_id`, `state_id`, `longitude`, `latitude`) VALUES
-(1, '1 bedroom', '15000000', 1, 1, '', ''),
-(2, '2 bedroom', '35000000', 1, 1, '', ''),
-(3, '1 bedroom', '20000000', 2, 1, '', ''),
-(4, '1 bedroom', '16000000', 1, 1, '0', '0');
+-- Table structure for table `houses_sale`
+--
+
+CREATE TABLE `houses_sale` (
```

Comment:
I think it is better to use a single table (`houses`) for rent and sale to data management easier. Let's merge the two tables together and separate them with the category column. For the search result page, we should only get data from the `For Sale` category.